community/crispy-doom/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+autoreconf -fi
```

Comment:
If upstream doesn't provide tarballs with this step already done, (you may want to ask them to), be sure to list `autoconf make` as a dependency.

Reply:
Thanks, good point! I'll see if i can prod upstream for this.